### DISCUSSION TITLE
Blank Apps and Users pages

### DISCUSSION THREAD

**Discussion by patrick-5546** [nextcloud/all-in-one#1299] [Questions] [answered]
I am unable to load the Apps and Users pages consistently. I have tried setting the file permissions of the data directory with `chmod 755 -R .` and running `occ maintenance:repair`, but this only fixes things temporarily. How can I make this persistent?

I installed nextcloud AIO on unraid following this guide: https://myunraid-ru.translate.goog/nextcloud-aio/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=de&_x_tr_pto=wapp#Cast_1_master-kontejner

**Comment by patrick-5546** [comment]
One person fixed it by running the update in CLI from the Unraid docker, but I'm not sure what command this would be: https://www.reddit.com/r/NextCloud/comments/ukpr6i/comment/i7sn28l/?utm_source=share&utm_medium=web2x&context=3

Another person said they fixed it by removing files from the /var/www/nextcloud directory hierarchy, but I'm not sure what that means: https://help.nextcloud.com/t/apps-and-users-pages-are-blank/71544/6

**Comment by szaimen** [comment, marked as answer]
Sorry, I cannot reproduce your issue.

Do you have some warnings in the Nextcloud overview and/or in the Nextcloud logs maybe?

**Reply by patrick-5546** [reply]
The problem seems to have resolved itself, I will add to this issue if I encounter it again.
